CHANGELOG.md

```diff
@@ -76,23 +78,23 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
   - Prover precomputations are moved into `ProverIndex`.
   - Helper `absorb_commitment` used by the prover and verifier to absorb shifted commitments in `PolyComm`.
 
-## [0.1.3] - 2022-11-27
+### [0.1.0] - 2022-11-27
 
 - **Changed**
   - Occurences of the Horner's method are replaced with the standard implementation in the `arkworks` crate.
 
-## [0.1.2] - 2022-11-25
+### [0.1.0] - 2022-11-25
 
 - **Changed**
   - `Fr_Sponge::absorb_evaluations` have deconstructed `ProofEvaluations`.
 
-## [0.1.1] - 2022-11-16
+### [0.1.0] - 2022-11-16
 
 - **Changed**: 
   - The generic gate is implemented using the `Argument` trait.
 
 
-## [0.1.0] - 2022-11-14
+### [0.1.0] - 2022-11-14
```

Comment:
I don't think it makes sense to have many headers for 0.1.0, and actually I think it'll trick parsers who are probably not expecting this (unless I misunderstand how changelogs should be formatted)

also I think just having "first version of kimchi" under 0.1.0 is enough? There's no "change" since there was nothing before. Perhaps we can move all of this to the unreleased part if we want :o

Reply:
yeah, that's a good example: https://keepachangelog.com/en/1.0.0/